### ISSUE TITLE
Does not work after last update

### ISSUE THREAD

**Issue by Vladisl0ve** [alexmon1989/russia_ddos#12]
Errors on the launching docker image (latest - 1.3.6) - so application does not work 

![image](https://user-images.githubusercontent.com/36882979/156900908-d225a362-d9c0-45cc-acd0-6551bfb04327.png)

Errors code:
```
File "/usr/local/lib/python3.10/http/client.py", line 1277, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/local/lib/python3.10/http/client.py", line 1037, in _send_output
    self.send(msg)
  File "/usr/local/lib/python3.10/http/client.py", line 975, in send
    self.connect()
  File "/usr/local/lib/python3.10/http/client.py", line 1447, in connect
    super().connect()
  File "/usr/local/lib/python3.10/http/client.py", line 941, in connect
    self.sock = self._create_connection(
  File "/usr/local/lib/python3.10/socket.py", line 824, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/usr/local/lib/python3.10/socket.py", line 955, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Try again

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/app/DRipper.py", line 592, in <module>
    sys.exit(main())
  File "/app/DRipper.py", line 576, in main
    show_info(_ctx)
  File "/app/DRipper.py", line 334, in show_info
    logo()
  File "/app/DRipper.py", line 252, in logo
    ╚═════╝ ╚═╝  ╚═╝╚═╝╚═╝     ╚═╝     ╚══════╝╚═╝  ╚═╝ ver.: {get_app_version()}
  File "/app/DRipper.py", line 97, in get_app_version
    response = urllib.request.urlopen(url).read()
  File "/usr/local/lib/python3.10/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/local/lib/python3.10/urllib/request.py", line 519, in open
    response = self._open(req, data)
  File "/usr/local/lib/python3.10/urllib/request.py", line 536, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/usr/local/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/local/lib/python3.10/urllib/request.py", line 1391, in https_open
    return self.do_open(http.client.HTTPSConnection, req,
  File "/usr/local/lib/python3.10/urllib/request.py", line 1351, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno -3] Try again>
```
P.s. Clear launching after pulling from Github without docker (via python3) makes now errors.

System: Ubuntu 21.04
Application version: latest (1.3.6)

**Comment by alexmon1989** [comment]
Please try again, we've fixed this bug (version 1.3.7).

**Comment by Vladisl0ve** [comment]
Good job
Works now :)
